generators/server/templates/src/main/resources/config/_application-dev.yml

```diff
@@ -34,8 +34,8 @@
 logging:
     level:
         ROOT: DEBUG
-        <%=packageName%>: DEBUG
         io.github.jhipster: DEBUG
+        <%=packageName%>: DEBUG
```

Comment:
the loggers must be in the correct hierarchical order to avoid mapping issues at startup. This is required for the sample projects running under package `io.github.jhipster.travis`